.github/workflows/CypressImage.yaml

```diff
@@ -0,0 +1,53 @@
+name: Cypress Build & Push
+env:
+  DATALAND_SKYMINDERCLIENT_TOKEN: ${{ secrets.DATALAND_SKYMINDERCLIENT_TOKEN }}
+  DATALAND_SKYMINDERCLIENT_USER: ${{ secrets.DATALAND_SKYMINDERCLIENT_USER }}
+  DATALAND_EDC_TOKEN: ${{ secrets.DATALAND_EDC_TOKEN }}
+  DATALAND_EDC_USER: ${{ secrets.DATALAND_EDC_USER }}
+  SKYMINDER_URL: ${{ secrets.SKYMINDER_URL }}
+  SKYMINDER_PW: ${{ secrets.SKYMINDER_PW }}
+  SKYMINDER_USER: ${{ secrets.SKYMINDER_USER }}
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+jobs:
+
+  build-and-push-docker-image:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'workflow_dispatch'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 17
+          cache: gradle
+      - name: Build all project outputs and npm install
+        run: ./gradlew assemble npmInstall --no-daemon
+      - name: Copy the gradle cache into the workspace
```

Comment:
statt ` Copy the gradle cache into the workspace`
bitte ` Copy the gradle and cypress cache into the workspace`

Reply:
Fixed

Reply:
ok